scalafix-core/src/main/scala-2/scalafix/internal/util/MetaconfigCompatMacros.scala

```diff
@@ -1,3 +1,4 @@
+
 package scalafix.internal.util
```

Comment:
It looks like https://github.com/scalacenter/scalafix/pull/1562 was only partially reverted - why? Thi seems to be used in the scala-2 version of `ConfigRulePatches` but should not longer be the case (so you won't need a separate scala-3 version).

For visibility, let's spin off the revert of https://github.com/scalacenter/scalafix/pull/1562 in a separate PR with the rationale (major clients have been released and scala-steward is back so the risk of using an old scalafix-cli with a rule compiled against a recent scalafix-core is limited).

Reply:
Here's the separate PR for the revert, feel free to edit the description if needed: https://github.com/scalacenter/scalafix/pull/1632

Reply:
merged, you can rebase